# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderTextureHost.cpp
# Commit: 72d4bf5b093e
# Full Hash: 72d4bf5b093e9b00eed78bf4e51850d56df5ae6c
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-05 16:09:03
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add RenderTextureHostSWGL support to D3D11 RenderTextureHosts. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95799
# ==============================================================================

diff -r 99b2e5c55a25 -r 72d4bf5b093e gfx/webrender_bindings/RenderTextureHost.cpp
--- a/gfx/webrender_bindings/RenderTextureHost.cpp	Thu Nov 05 09:14:52 2020 +0000
+++ b/gfx/webrender_bindings/RenderTextureHost.cpp	Thu Nov 05 09:15:05 2020 +0000
@@ -49,6 +49,7 @@
 
 wr::WrExternalImage RenderTextureHost::LockSWGL(uint8_t aChannelIndex,
                                                 void* aContext,
+                                                RenderCompositor* aCompositor,
                                                 wr::ImageRendering aRendering) {
   return InvalidToWrExternalImage();
 }